src/main/java/org/gridsuite/directory/server/dto/ElementAttributes.java

```diff
@@ -40,6 +41,10 @@
 
     private String description;
 
+    public boolean isAllowed(@NonNull String userId) {
+        return owner.equals(userId) || !accessRights.isPrivate();
```

Comment:
it's scary to have this method here. It works fine if we call it on a directory element but it doesn't in any other case as we want to know the accessRights of the parent

Reply:
I agree but we don't have a dto dedicated to a folder (as for entities)

Reply:
Control added